Create new table for location specific latest analytics.

LATaylor-guardian opened the pull request:
This will eventually replace the previous table which will be deleted on switchover.

janua commented on cloudformation/dynamoDb.yaml:
```diff
@@ -240,6 +240,32 @@ Resources:
+      KeySchema:
+      - AttributeName: territory
+        KeyType: HASH
+      - AttributeName: campaignId
```
Which way around is better, is this going to be displayed inside a particular campaign? If so maybe the opposite might be better?

LATaylor-guardian replied:
@janua I thought about this for a bit. I think this way as Campaign Central will be contextual to the territory which the user will select at the top level of Campaign Central. So this allows you to query for all campaigns in a given territory.

janua replied:
Ah I see... Then you can batch get the campaigns!

LATaylor-guardian replied:
Yeah :)

janua commented:
:+1:

Commit:
Create new table for location specific latest analytics.

This will eventually replace the previous table which will be deleted on switchover.
